corehq/apps/users/models.py

```diff
@@ -230,6 +230,11 @@ class UserRole(Document):
     permissions = SchemaProperty(Permissions)
     is_archived = BooleanProperty(default=False)
 
+    @classmethod
+    @quickcache(['doc_id'], timeout=0)
+    def get(cls, doc_id):
```

Comment:
yeah i do agree with these critiques of `CachedCouchDocumentMixin` and your note on the other PR that it should ideally be split between the thing that tries to cache `.get` and the thing that tries to cache `.view`. I think if we wanted to do this type of caching here then using that same mechanism could be better/easier, and that way when we do go clean up / improve the behavior of `CachedCouchDocumentMixin` it will be standardized. also fine with the simpler "change the name" approach too.